cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py

```diff
@@ -50,7 +50,18 @@ class CxTowerCommandExecuteWizard(models.TransientModel):
         help="Run commands using 'sudo'",
     )
     code = fields.Text(compute="_compute_code", readonly=False, store=True)
-    any_server = fields.Boolean(default=True)
+    applicability = fields.Selection(
+        selection=[
+            ("this", "For selected server(s)"),
```

Comment:
```suggestion
            ("this", "For server(s) filtered by tags"),
```
or something like that, because there is no actual selection of servers, only filtering by tags, it can be confusing.

<img width="984" alt="Screenshot 2025-02-25 at 11 57 19" src="https://github.com/user-attachments/assets/a2454a02-48d8-4b0e-83f0-916348d94b33" />

Reply:
I got your point, however we filter commands, not servers. 
Need to refactor this.